templates/main/more_builds.html.ep

```diff
@@ -7,3 +7,12 @@
     % my $selected = $rtagged{$only_tagged};
     %= b join(' / ', map { $_ eq $selected ? "<b>$_</b>" : link_to($_ => url_with->query([only_tagged => $tagged{$_}])) } reverse sort keys %tagged);
 </div>
+
```

Comment:
@okurz after you pointed out the "limit to" section, i made the full screen mode part of it
![screenshot_2017-08-21_14-41-18](https://user-images.githubusercontent.com/22001671/29519921-6f53afa8-867f-11e7-97b9-249152284f52.png)